package.json

```diff
@@ -127,27 +125,29 @@
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-react": "^7.9.1",
     "eslint-plugin-standard": "^3.1.0",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "flow-bin": "^0.65.0",
     "mocha": "~5.2.0",
     "nodemon": "^1.17.2",
     "onchange": "^4.0.0",
     "prettier": "^1.13.4",
     "proxyquire": "~1.8.0",
     "sinon": "~2.3.8",
-    "sinon-chai": "~2.12.0"
+    "sinon-chai": "~2.12.0",
+    "webpack-cli": "^3.0.8"
   },
   "license": "MIT",
   "scripts": {
     "start": "node app.js",
     "dev": "concurrently --kill-others \"npm run dev:server\" \"npm run dev:client\" \"npm run watch\" ",
     "dev:server": "DEBUG=crowi:* NODE_ENV=development nodemon --watch ./lib/ --watch ./locales/ --ext 'js,json' --exec node app.js",
-    "dev:client": "NODE_ENV=development webpack --progress --profile --colors --display-error-details --watch --config tools/webpack.config.js",
+    "dev:client": "NODE_ENV=development webpack --progress --profile --colors --display-error-details --watch --mode=development --config tools/webpack.config.js",
     "flow": "flow",
     "format": "eslint --fix \"**/*.js\"",
     "lint": "eslint \"**/*.js\"",
     "test": "mocha -r test/bootstrap.js --globals chai --reporter dot test/**/*.test.js",
     "build": "npm run prod:client",
-    "prod:client": "NODE_ENV=production webpack --bail --progress --colors --display-error-details --config tools/webpack.config.js",
+    "prod:client": "NODE_ENV=production webpack --bail --progress --colors --display-error-details --mode=production --config tools/webpack.config.js",
```

Comment:
I removed `--progress` on #313 :)

Reply:
Thanks for letting me know.
Since `mode` option refers to an environment variable, merge conflict does not occur now.